### DISCUSSION TITLE
Convert EBSDsquare to arbitrarily aligned EBSD

### DISCUSSION THREAD

**Discussion by ghost** [mtex-toolbox/mtex#748] [Ask Anything] [answered]
Hello all,

I have been processing some large-body EBSD data and using the function of _fill_. Before filling, the EBSD data is arbitrarily aligned (e.g., 20000 * 1 EBSD). After filling the EBSD data becomes a matrix, with "m * n EBSDsquare". 

My question is that is there a way to convert the EBSDsquare back to arbitrarily aligned EBSD?

Many thanks,
Sheng

**Comment by kilir** [comment, marked as answer]
Hi Sheng,

both, `ebsd(:)` or `EBSD(ebsd)` will give you the ebsd data again as a list.

Cheers,
Rüdiger

**Reply by ghost** [reply]
Thank you so much Rüdiger!

Cheers,
Sheng
